### PR TITLE
Refactor add user

### PR DESCRIPTION
**Pull request by henrikstranneheim**
## Description

Refactor `add_user`

### Added

- `user_filter.py`
- Tests

### Changed

- Moved add_user function to UpdateHandler

### How to prepare for test
- [ ] ssh to hasta (depending on type of change)
- [ ] activate stage: `us`
- [ ] request trailblazer-stage on hasta: `paxa`
- [ ] install on stage:
    ```Shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_trailblazer -t trailblazer -b refactor_add_user -a
    ```
- [ ] ssh to clinical-db (depending on type of change)
- [ ] install on stage:
`bash /home/proj/production/servers/resources/clinical-db.scilifelab.se/update-trailblazer-ui-stage.sh [THIS-BRANCH-NAME]`

### How to test
- [ ] login to ...
- [ ] do ...

### Expected test outcome
- [ ] check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review
- [ ] tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: tests/store/filters/test_user_fitlers.py
Comment (seallard) [inline tests/store/filters/test_user_fitlers.py:1]: Typo in file name

**Review by seallard — APPROVED** [review]
Great 👍

File: tests/store/filters/test_user_fitlers.py
Comment (henrikstranneheim) [reply tests/store/filters/test_user_fitlers.py:1]: Thx! Fixed

**Comment by henrikstranneheim** [comment]
Deployed on hasta:
```
Log deploy... done.
trailblazer, version 12.0.14
```

**Pull request merged**